tutoraspects/templates/openedx-assets/queries/dim_at_risk_learners.sql

```diff
@@ -0,0 +1,30 @@
+with
+    page_visits as (
+        select org, course_key, actor_id, max(last_visited) as last_visited
+        from {{ ASPECTS_XAPI_DATABASE }}.fact_learner_last_course_visit
+        where
+            1=1
+            {% include 'openedx-assets/queries/common_filters.sql' %}
+        group by org, course_key, actor_id
+    )
```

Comment:
Same as above, usually the MV with org/course_key partition is good enough to improve the query times